src/aws.ts

```diff
@@ -504,6 +507,237 @@ export namespace $AWS {
       },
     });
   }
+
+  export const SDK: StepFunctionSDK = new Proxy<any>(
+    {},
+    {
+      get(_, serviceName: ServiceKeys) {
+        return new Proxy<any>(
+          {},
+          {
+            get: (_, methodName: string) => {
+              const defaultServicePrefix = serviceName.toLowerCase();
```

Comment:
wouldn't this require similar logic to the `mapAslSdkServiceName` method?

Reply:
I added an `IAM_SERVICE_PREFIX` mapping in `iam.ts` and kept the `.toLowerCase()` as a fallback. 👍  

The StepFunctions service name mapping only had a handful (~15) of exceptions to the `.toLowerCase()` "rule". IAM prefixes on the other hand are a lot messier. Out of over 310 service classes in the sdk only about 190 follow this rule. So currently I made the whole mapping explicit, but if you want to we can get rid of the 190 lines that follow this rule.